sdk/nodejs/runtime/closure/codePaths.ts

```diff
@@ -106,52 +106,55 @@ export async function computeCodePaths(
 async function computeCodePathsWorker(options: CodePathOptions): Promise<Map<string, asset.Asset | asset.Archive>> {
     // Construct the set of paths to include in the archive for upload.
 
-    // Find folders for all packages requested by the user
-    const pathSet = await allFoldersForPackages(
+    // Find folders for all packages requested by the user.  Note: all paths in this should
+    // be normalized.
+    const normalizedPathSet = await allFoldersForPackages(
         new Set<string>(options.extraIncludePackages || []),
         new Set<string>(options.extraExcludePackages || []),
         options.logResource);
 
     // Add all paths explicitly requested by the user
     const extraIncludePaths = options.extraIncludePaths || [];
     for (const path of extraIncludePaths) {
-        pathSet.add(path);
+        // Note: use 'normalizeSafe' so that we don't unintentionally strip off "./" from the
```

Comment:
Why do we not want to do this? It seems to me that `./node_modules/foo/bar` and `node_modules/foo/bar` should be considered the same?

Reply:
I think you're riht. have switched back.